### PR TITLE
Next release

### PR DESCRIPTION
**Pull request by vanGalilea**
## 🎨 UI Changes

<details>
<summary>iOS</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

<details>
<summary>Android</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

## ☑️ Checklist

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] PR targets the `develop` branch
- [ ] Commits follow the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) spec
- [ ] New code is covered by tests
- [ ] Screenshots added for visual changes
- [ ] Documentation is updated

### CODE REVIEW

File: examples/SampleApp/android/build.gradle [hunk 1]
```diff
@@ -42,6 +42,7 @@ allprojects {
             }
         }
         google()
+        jcenter()
```
Comment (madsroskar) [inline examples/SampleApp/android/build.gradle:45]: Isn't jcenter [discontinued](https://developer.android.com/studio/build/jcenter-migration)? Should we look into alternatives for this?

File: examples/TypeScriptMessaging/android/build.gradle [hunk 2]
```diff
@@ -38,6 +38,7 @@ allprojects {
             }
         }
         google()
+        jcenter()
```
Comment (madsroskar) [inline examples/TypeScriptMessaging/android/build.gradle:41]: See other comment about jcenter

File: package/native-package/src/index.js [hunk 3]
```diff
@@ -247,6 +249,24 @@ registerNativeHandlers({
       ignoreAndroidSystemSettings: false,
     });
   },
+  // eslint-disable-next-line react/display-name
```
Comment (madsroskar) [inline package/native-package/src/index.js:252]: Should we add a task to add a displayname to the video component?

**Review by madsroskar — COMMENTED** [review]
We should make sure we want to release this using jcenter. @khushal87 did you see if there are any alternatives to this?

File: examples/TypeScriptMessaging/android/build.gradle (hunk 2 above)
Comment (vanGalilea) [reply examples/TypeScriptMessaging/android/build.gradle:41]: This is NOT blocking for the release: @khushal87 please investigate and remove/replace

File: package/src/components/Attachment/Gallery.tsx [hunk 4]
```diff
@@ -258,35 +279,34 @@ const GalleryWithContext = <
...
+                    />
+                  ) : (
+                    <MemoizedGalleryImage
+                      resizeMode={resizeMode}
+                      style={[
+                        borderRadius,
+                        image,
+                        {
+                          height: height - 1,
+                          width: width - 1,
```
[the remark this answers is not in the document]
Comment (khushal87) [reply package/src/components/Attachment/Gallery.tsx:301]: Not sure @vishalnarkhede can you help here?

File: examples/TypeScriptMessaging/android/build.gradle (hunk 2 above)
Comment (khushal87) [reply examples/TypeScriptMessaging/android/build.gradle:41]: I don't remember how I introduced this but as far as I know removing this crashed the app on the build. I will investigate it further.

File: package/native-package/src/index.js (hunk 3 above)
Comment (vanGalilea) [reply package/native-package/src/index.js:252]: As Mads said, this is NOT blocking for the release: @khushal87 let's solve this by creating a const arrow func or any other valid solution besides suppressing the linter

File: package/src/components/Attachment/Gallery.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (vanGalilea) [reply package/src/components/Attachment/Gallery.tsx:301]: This is NOT blocking for the release: @khushal87 let's fix this with a margin/padding, anything beside this -1 hack :|

File: examples/TypeScriptMessaging/android/build.gradle (hunk 2 above)
Comment (khushal87) [reply examples/TypeScriptMessaging/android/build.gradle:41]: Error without `jcenter()`:

```
> Task :react-native-video:generateDebugRFile FAILED

Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/7.2/userguide/command_line_interface.html#sec:command_line_warnings
192 actionable tasks: 2 executed, 190 up-to-date

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-video:generateDebugRFile'.
> Could not resolve all files for configuration ':react-native-video:debugCompileClasspath'.
   > Could not find com.google.android.exoplayer:exoplayer:2.13.2.
     Required by:
         project :react-native-video
   > Could not find com.google.android.exoplayer:extension-okhttp:2.13.2.
     Required by:
         project :react-native-video

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 32s

error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.
Error: Command failed: ./gradlew app:installDebug -PreactNativeDevServerPort=8081

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-video:generateDebugRFile'.
> Could not resolve all files for configuration ':react-native-video:debugCompileClasspath'.
   > Could not find com.google.android.exoplayer:exoplayer:2.13.2.
     Required by:
         project :react-native-video
   > Could not find com.google.android.exoplayer:extension-okhttp:2.13.2.
     Required by:
         project :react-native-video

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 32s

    at makeError (/Users/khushalagarwal/Projects/stream-chat-react-native/examples/SampleApp/node_modules/execa/index.js:174:9)
    at /Users/khushalagarwal/Projects/stream-chat-react-native/examples/SampleApp/node_modules/execa/index.js:278:16
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async runOnAllDevices (/Users/khushalagarwal/Projects/stream-chat-react-native/examples/SampleApp/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/runOnAllDevices.js:109:5)
    at async Command.handleAction (/Users/khushalagarwal/Projects/stream-chat-react-native/examples/SampleApp/node_modules/@react-native-community/cli/build/index.js:192:9)
```

Solution used: https://github.com/react-native-video/react-native-video/issues/2468#issuecomment-913709636

File: package/src/components/Attachment/Gallery.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (khushal87) [reply package/src/components/Attachment/Gallery.tsx:301]: This comes from - https://github.com/GetStream/stream-chat-react-native/pull/1056/files#diff-e7d0e211073121199347eedf392123d964951dbd5ebbda17d1f86c921ec39782R308

I need to be sure if it's used for Margin. I will have a chat with Vishal about this and update it.

**Review by vishalnarkhede — APPROVED** [review]

**Pull request merged**